### PR TITLE
chore: add volunteer hartpoli

### PR DESCRIPTION
**Pull request by hartpoli**
added

### CODE REVIEW

File: data/users/hartpoli.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "bio": "ML | Web | Android  ",
+  "image": "https://avatars.githubusercontent.com/u/65838275?s=400&v=4",
+  "categories": [
+    {
+      "slug": "programming | software-development | web-development | mobile-development | machine-learning",
+      "topics": []
+    }
+  ],
+  "contacts": {
+    "twitter": ""
```
Comment (codewithvoid) [inline data/users/hartpoli.json:12]: add your twitter handle here for people to tweet questions at you

File: data/users/hartpoli.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "hartpoli",
+  "bio": "ML | Web | Android  ",
+  "image": "https://avatars.githubusercontent.com/u/65838275?s=400&v=4",
+  "categories": [
+    {
+      "slug": "programming | software-development | web-development | mobile-development | machine-learning",
```
Comment (codewithvoid) [inline data/users/hartpoli.json:7]: per slug entry you need to pick 1 topic.

Also mention specific topics you want to help other developers with. E.g NodeJS, TailwindCSS etc. 
Refer other data files.

File: data/users/hartpoli.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "hartpoli",
+  "bio": "ML | Web | Android  ",
```
Comment (codewithvoid) [inline data/users/hartpoli.json:3]: pick 1 bio. E.g Web Developer.

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thank you @hartpoli for volunteering. Added some comments. Please check.

File: data/users/hartpoli.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  "bio": "Backend Developer ",
+  "image": "https://avatars.githubusercontent.com/u/65838275?s=400&v=4",
+  "categories": [
+    {
+      "slug": "NodeJs",
+      "topics": ["Express"]
+    },
+    {
+      "slug": "programming",
+      "topics": ["JavaScript", "Python", "C+"]
```
Comment (codewithvoid) [inline data/users/hartpoli.json:12]: nit: C++

File: data/users/hartpoli.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "hartpoli",
+  "bio": "Backend Developer ",
+  "image": "https://avatars.githubusercontent.com/u/65838275?s=400&v=4",
+  "categories": [
+    {
+      "slug": "NodeJs",
```
Comment (codewithvoid) [inline data/users/hartpoli.json:7]: Update this as below:
```json
"slug": "web-development",
"topics": ["NodeJS", "ExpressJS"]
```

File: data/users/hartpoli.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+      "topics": ["Express"]
+    },
+    {
+      "slug": "programming",
+      "topics": ["JavaScript", "Python", "C+"]
+    }
+  ],
+  "contacts": {
+    "twitter": "https://twitter.com/yhartpx123"
```
Comment (codewithvoid) [inline data/users/hartpoli.json:16]: just username is ok.
```json
"twitter": "yhartpx123"
```

**Review by codewithvoid — CHANGES_REQUESTED** [review]

**Comment by hartpoli** [comment]
anyinfo for update?

File: data/users/hartpoli.json [hunk 7]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "hartpoli",
+  "bio": "Backend Developer ",
+  "image": "https://avatars.githubusercontent.com/u/65838275?s=400&v=4",
+  "categories": [
+    {
+      "slug": "Web Development",
```
Comment (codewithvoid) [inline data/users/hartpoli.json:7]: slug doesn't have space and is lowercase
```
      "slug": "web-development",
```

File: data/users/hartpoli.json [hunk 8]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "hartpoli",
+  "bio": "Backend Developer ",
```
Comment (codewithvoid) [inline data/users/hartpoli.json:3]: remove extra space in the end

**Review by codewithvoid — CHANGES_REQUESTED** [review]

**Review by codewithvoid — APPROVED** [review]
Thanks for making the changes. Welcome.

You can join community discord [here]([discord.gg/DFwsFKfgEB](https://t.co/CyAqwr7LNo)).

**Comment by hartpoli** [comment]
thank u..

**Pull request merged**